openrtb_ext/bidders.go

```diff
@@ -568,10 +565,42 @@ type BidderParamValidator interface {
 	Schema(name BidderName) string
 }
 
+type paramsValidatorHelper interface {
```

Comment:
Better to name it more specific, something like `bidderOS`, `bidderFileSystem`, or something better which directly ties this being used to discover bidders defined in the file system.

Reply:
https://github.com/prebid/prebid-server/pull/3048/commits/8e4cd4c9252d8b836b76b0833cb201c24d58aa3a